### PR TITLE
Stub out more basic NA Summit info

### PR DESCRIPTION
**Pull request by jeefy**
Mainly a copy/pasta with some updated verbage and info. :)

### CODE REVIEW

File: events/2019/11-contributor-summit/README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+## Registration
+
+- [Sign the CLA](/CLA.md) if you have not done so already.
+- Registration Form Coming Soon
+
+## When and Where
+
+The Contributor Summit takes place in the days leading up to [KubeCon + CloudNativeCon NA](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019/), make sure you plan travel accordingly.
+
+- Sunday, November 17th, 2019 - Social event (optional)
```
Comment (parispittman) [inline events/2019/11-contributor-summit/README.md]: put a tbd time range so people know when to plan flights. something like ~5pm-8pm tba

File: events/2019/11-contributor-summit/README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+- Monday, November 18th, 2019 - Contributor Summit (New Contributor Workshops and Current Contributor Summit)
```
Comment (parispittman) [inline events/2019/11-contributor-summit/README.md]: same as above - do ~8am-6pm tba

File: events/2019/11-contributor-summit/README.md [hunk 3]
```diff
@@ -1 +1,50 @@
...
+- Content Sharing: All presentations will be uploaded.
+
+## Team
+
+| Role | Lead | Shadow | Notes |
+|---|---|---|---|
+| Event Lead | Paris | Jeff | |
+| Project Manager | Jamie | himanshu | |
+| Registration | Bob | Nabarun | |
+| Marketing | Jonas | * | |
```
Comment (parispittman) [inline events/2019/11-contributor-summit/README.md]: nzoueidi for marketing shadow

File: events/2019/11-contributor-summit/README.md [hunk 4]
```diff
@@ -1 +1,50 @@
...
+- [Sign the CLA](/CLA.md) if you have not done so already.
+- Registration Form Coming Soon
+
+## When and Where
+
+The Contributor Summit takes place in the days leading up to [KubeCon + CloudNativeCon NA](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019/), make sure you plan travel accordingly.
+
+- Sunday, November 17th, 2019 - Social event (optional)
+- Monday, November 18th, 2019 - Contributor Summit (New Contributor Workshops and Current Contributor Summit)
+- San Diego, California
```
Comment (parispittman) [inline events/2019/11-contributor-summit/README.md]: add a line about the fun doesn't stop there - kubecon tu-th maintainer track (SIG Intros, Deep Dives, and contributor talks about upstream adventures)

File: events/2019/11-contributor-summit/README.md [hunk 5]
```diff
@@ -1 +1,50 @@
...
+| New Contributor Workshop | Guin | Rael and/or Allison | Dawn Foster to organize SIG Meet and Greet |
+| Accessibility, Inclusiveness, and Diversity | Tasha | Arnaud | |
+
+## Code of Conduct
+
+This event, like all Kubernetes events, has a [Code of Conduct](/code-of-conduct.md). We will have an onsite rep with contact information to be provided here and posted during the event.
+
+## Misc
+
+We want to remove as many barriers as possible for you to attend this event. Please contact community@kubernetes.io to see if we can accommodate a request.
```
Comment (parispittman) [inline events/2019/11-contributor-summit/README.md]: add #summit-staff as a contact point for folks to reach us or general info and hangout at #contributor-summit

File: events/2019/11-contributor-summit/README.md (hunk 1 above)
Comment (jeefy) [reply events/2019/11-contributor-summit/README.md]: Done :)

File: events/2019/11-contributor-summit/README.md (hunk 3 above)
Comment (jeefy) [reply events/2019/11-contributor-summit/README.md]: Done :)

File: events/2019/11-contributor-summit/README.md (hunk 2 above)
Comment (jeefy) [reply events/2019/11-contributor-summit/README.md]: Done :)

File: events/2019/11-contributor-summit/README.md (hunk 4 above)
Comment (jeefy) [reply events/2019/11-contributor-summit/README.md]: BUT WAIT THERE'S MORE. :)

File: events/2019/11-contributor-summit/README.md (hunk 5 above)
Comment (jeefy) [reply events/2019/11-contributor-summit/README.md]: Done, although I'm curious if we should link to #summit-staff at all. Realistically we address questions in the #contributor-summit channel already. 

Thoughts?

**Comment by jeefy** [comment]
Addressed suggestions, waiting on one question to be answered then I can squashy squashy. :)

File: events/2019/11-contributor-summit/README.md (hunk 5 above)
Comment (parispittman) [reply events/2019/11-contributor-summit/README.md]: 100% ok - squash and go :)

**Comment by jeefy** [comment]
Squashed and ready. :)

**Comment by parispittman** [comment]
/approve

will leave the lgtm to josh or someone

**Pull request merged**